### DISCUSSION TITLE
running wsl --update with no admin privilege : is it a valid install ?

### DISCUSSION THREAD

**Discussion by 2pl** [microsoft/WSL#9955] [Q&A]
I'm running wsl2 on a corporate computer.
I do not have administrator rights on this computer.
Recently  i ran 
`wsl --update`
- it proceeded with download
- then at around 90% completion i had a popup asking for administrator account
- i clicked cancel on this popup as i have no admin account
- the update completed successfully though (at least it said so), and i now have:
```
wsl --version
WSL version: 1.2.0.0
Kernel version: 5.15.90.1
WSLg version: 1.0.51
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.2728
``` 
So i now have an updated kernel, but still concerned that some steps of the upgrade might have failed given i did not provide admin account. Any view on this ?
The reason i'm asking is i'm seeing frequent crashes since i updated, all my wsl instances seem to simultaneously stop with:
`[process exited with code 1 (0x00000001)]
`
Usually happens after I close my laptop / change location and network / reopen it
I have this type of errors in eventlog at the time of crashes:
```
Faulting application name: wslservice.exe, version: 1.2.0.0, time stamp: 0x642c5a52
Faulting module name: ntdll.dll, version: 10.0.19041.2130, time stamp: 0xb5ced1c6
Exception code: 0xc000000d
Fault offset: 0x0000000000112684
Faulting process ID: 0x424
Faulting application start time: 0x01d96d359bfaf91a
Faulting application path: C:\Program Files\WindowsApps\MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.0.0_x64__8wekyb3d8bbwe\wslservice.exe
Faulting module path: C:\WINDOWS\SYSTEM32\ntdll.dll
Report ID: c951ddfa-864f-44eb-89fe-5de5275f9b83
Faulting package full name: MicrosoftCorporationII.WindowsSubsystemForLinux_1.2.0.0_x64__8wekyb3d8bbwe
Faulting package-relative application ID: wsl

```

**Comment by 2pl** [comment]
Created a bug from this since i have very frequent crashes.

**Reply by pkeir** [reply]
Same for me. Can you share the link to your bug?
